tools/tests/utils.py

```diff
@@ -97,6 +98,8 @@ def queries_from_config(config_path):
                     packcontent = rmcomment.sub('', packfile)
                     packqueries = json.loads(packcontent)
                     for queryname, query in packqueries["queries"].items():
+                        if "platform" in query and query['platform'] != spec_platform:
+                            continue
                         queries["pack_" + queryname] = query["query"]
```

Comment:
I'm not totally up on this code, but doesn't `query["query"]` include the platform key? It seems like it would make more sense to skip this in the `profile` routine.

Reply:
no, the platform was removed - only the actual SQL is retrieved from ```query["query"]```

```
(Pdb) n
> /Users/jasonlau/Software/osquery/tools/tests/utils.py(100)queries_from_config()
-> queries["pack_" + queryname] = query["query"]
(Pdb) p queryname
'schedule'
(Pdb) p query
{'query': 'select name, interval, executions, output_size, wall_time, (user_time/executions) as avg_user_time, (system_time/executions) as avg_system_time, average_memory, last_executed, denylisted from osquery_schedule;', 'interval': 7200, 'removed': False, 'denylist': False, 'version': '2.11.0', 'description': 'Report performance for every query within packs and the general schedule.'}
(Pdb) p query["query"]
'select name, interval, executions, output_size, wall_time, (user_time/executions) as avg_user_time, (system_time/executions) as avg_system_time, average_memory, last_executed, denylisted from osque
```